src/index.js

```diff
@@ -1 +1,42 @@
 import './index.css';
+
+const taskListContainer = document.getElementById('todoList');
+const tasks = [
+  {
+    description: 'complete',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'To do something',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'list tasks',
+    completed: true,
+    index: 3,
+  },
+];
+
+export default class DisplayTasks {
+  static renderTasks() {
+    tasks.sort((a, b) => a.index - b.index);
+    taskListContainer.innerHTML = '';
+    tasks.forEach((task, index) => {
+      taskListContainer.innerHTML += `
+        <li class="task" draggable="true" data-index="${index}">
+          <div class="checkbox-container">
+            <input type="checkbox" name="${task.description}" ${task.completed ? 'checked' : ''}>
+            <input type="text" value="${task.description}" readonly>
+          </div>
+          <i class="fas fa-ellipsis-vertical" data-index="${index}"></i>
+        </li>
+      `;
+    });
+  }
+}
```

Comment:
- [ ] Your styling looks different from the one required. Kindly syle your structure to suit the wireframe layout.
  
  ![image](https://github.com/axcell-0/todo-list/assets/81468675/2497c174-f4de-41d7-a18a-0f3fe0e3f7cc)

  ![kdkd](https://github.com/axcell-0/todo-list/assets/81468675/f3b94e85-9664-4f6d-acd5-1c458f56766a)